pkg/network/netlink/conntrack.go

```diff
@@ -47,12 +47,18 @@ type conntrack struct {
 }
 
 func (c *conntrack) Exists(conn *Con) (bool, error) {
+	var family byte = unix.AF_INET
+	if (conn.Con.Origin != nil && conn.Con.Origin.Src != nil && conn.Con.Origin.Src.To4() == nil) ||
```

Comment:
Do you know what family we need to use for IPv6-mapped IPv4 addresses? I ask because `To4()` will convert to IPv4 in that case. Alternatively you can use `len(conn.Con.Origin.Src) == net.IPv6len`

Reply:
Actually in the units tests, the IPv4 IPs passed in here are IPv6 mapped, and the tests passed before with `AF_INET`; this will continue to be the case when using `To4()`. I did start with using `len`, but that was failing for the IPv4 tests.

Reply:
I'm not seeing the mapped addresses being used. Can you point me to them?